CMakeLists.txt

```diff
@@ -145,6 +150,29 @@ if(SDL2NET_BUILD_SHARED_LIBS)
     sdl_target_link_options_no_undefined(SDL2_net)
 endif()
 
+if(SDL2NET_BUILD_APPLE_FRAMEWORK)
+    set(RESOURCE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/Xcode/pkg-support/resources/CMake" "COPYING.txt")
+    file(GLOB FRAMEWORK_CMAKE_CONFIG_FILES "${CMAKE_CURRENT_SOURCE_DIR}/Xcode/pkg-support/resources/CMake/*")
```

Comment:
You can't re-use the Xcode CMake files. These only provide a `SDL2::SDL2` target for the shared library.

You need to generate and install `SDL2ConfigVersion.cmake` and  `SDL2Config.cmake` + do `install(EXPORT)` .
With some refactoring, that code can be shared with unix prefix and framework prefixes.